chemprop/cli/hpopt.py

```diff
@@ -43,16 +43,16 @@
     DEFAULT_SEARCH_SPACE = {
         "activation": tune.choice(categories=list(Activation.keys())),
         "aggregation": tune.choice(categories=list(AggregationRegistry.keys())),
-        "aggregation_norm": tune.quniform(lower=1, upper=200, q=1),
-        "batch_size": tune.loguniform(lower=16, upper=256, base=2),
-        "depth": tune.quniform(lower=2, upper=6, q=1),
+        "aggregation_norm": tune.qrandint(lower=1, upper=200, q=1),
+        "batch_size": tune.choice(np.logspace(start=4, stop=8, num=5, base=2)),
```

Comment:
```suggestion
        "batch_size": tune.choice(np.logspace(start=4, stop=8, num=5, base=2, dtype=int)),
```

Then I don't think you need to set it to an int down in update_args_with_config. I could be wrong about that if the number is still read as a float from the config file

Reply:
Good catch!
Implementing this.